src/domain/app/i18n/sv.json

```diff
@@ -773,6 +782,8 @@
       "labelStartTime": "Starttid",
       "labelTime": "Tid",
       "labelUmbrellaEvent": "Huvudevenemang",
+      "labelUserConsent": "I have read <a href=\"https://www.hel.fi/static/liitteet-2019/Kaupunginkanslia/Rekisteriselosteet/Keha/Yritystoiminnan%20tukeminen%20ja%20neuvonta%20seka%20matkailuneuvontapalvelut.pdf\" target=\"_blank\" rel=\"noopener noreferrer\" aria-label=\"Matkailu register statement {{openInNewTab}}\">Matkailu register statement</a> and consent to use of my information",
```

Comment:
Should this be in Swedish?

Reply:
Translations: https://github.com/City-of-Helsinki/linkedcomponents-ui/pull/185/commits/9d8ee2af93331ff695811a6a1d684ae18200167c